server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueue.java

```diff
@@ -82,7 +86,10 @@ public Mail getMail() {
         @Override
         public void done(boolean success) throws MailQueueException {
             try {
-                rabbitClient.ack(deliveryTag);
+                if (success) {
```

Comment:
if false, shouldn't we rollback ?

Reply:
Handled here https://github.com/linagora/james-project/pull/1728/commits/0aeb0baf043a6bc40be74b17557d0ec74a44f66f